docs/source/_static/data_processing_examples/3d_smFISH.py

```diff
@@ -4,6 +4,15 @@
 
 This notebook walks through a work flow that analyzes one field of view of a mouse gene panel from
 the Allen Institute for Cell Science, using the starfish package.
+
```

Comment:
this is a nit, but all our notebooks are named <assay acronym>.ipyb/py. this one is not. what if we do 3d_smFISH -> either smFISH. or smFISH_3d. i don't like making a big deal out of 3D for just one notebook, especially when not all our stuff works in 3D yet, e.g., segmentation and target assignment for example.

Reply:
Also, this isn't from the Allen Institute for Cell Science

Reply:
lol. nice catch. Cell Science -> Brain Science. Brains do have cells though i think.